gnuplot/Dockerfile

```diff
@@ -1,3 +1,14 @@
+# Run gnuplot in a container
```

Comment:
It would be even more useful to move this comment into an actual, working `run.sh` file in the same directory (add `"$@"` after the final "jess/gnuplot").